### ISSUE TITLE
Docs under Concepts/Peripherals is incomplete

### ISSUE THREAD

**Issue by dufguix** [tinygo-org/tinygo-site#428]
Hi,

It could be interesting to find the docs about ADC, SPI, UART, PWM under Concepts>Peripherals alongside GPIO, I2C, Watchdog.

Cheers

**Comment by aykevl** [comment]
Yes, it's still missing. The goal was to write the documentation gradually, but we haven't finished it yet.
